app/templates/email/bug-report.ts

```diff
@@ -1,6 +1,6 @@
 export const bugReportTemplates = {
   en: `
-Describe the bug
+Describe the bug (https://cube-validator.lindas.admin.ch/select)
```

Comment:
@noahonyejese I meant to have it next to `Cube Validator` string, not in the title 😄👍

Reply:
oh sorry misunderstood that